src/main/java/edu/wpi/grip/Main.java

```diff
@@ -1,5 +1,33 @@
 package edu.wpi.grip;
 
-public class Main {
-    public static void main(String[] args) {}
+import javafx.application.Application;
+import javafx.beans.value.ObservableValue;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Cursor;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Screen;
+import javafx.stage.Stage;
+
+public class Main extends Application {
```

Comment:
Presumably headless mode would have a different main class.  We'll add then when we have a headless mode.